include/unk_02031734.h

```diff
@@ -0,0 +1,215 @@
+#ifndef POKEDIAMOND_UNK_02031734_H
+#define POKEDIAMOND_UNK_02031734_H
+
+#include "global.h"
+#include "MI_memory.h"
+#include "game_init.h"
+#include "heap.h"
+#include "player_data.h"
+#include "unk_0202F150.h"
+
+struct UnkStruct_02031734_const1
+{
+    u32 unk00[3];
```

Comment:
do these work as void *

Reply:
No because the size of the structs is important as they get copied to the stack in the functions they are used. I tried to remove these structs altogether and create the arrays directly in the functions but they ended up in the wrong order in .rodata